tests/conftest.py

```diff
@@ -15,7 +15,13 @@
 import isal.isal_zlib
 import pytest
 import zlib_ng.zlib_ng
-from blockbuster import blockbuster_ctx
+
+try:
+    from blockbuster import blockbuster_ctx
+
+    have_blockbuster = True
+except ImportError:
+    have_blockbuster = False
```

Comment:
Consistency with other modules:
```suggestion
    HAS_BLOCKBUSTER = True
except ImportError:
    HAS_BLOCKBUSTER = False
```

Reply:
Implemented in a follow-up commit; thanks.